ci/pipelines/infra-drift-detector-lower-envs.yml

```diff
@@ -0,0 +1,145 @@
+resource_types:
+  - name: slack-notification
+    type: docker-image
+    source:
+      repository: cfcommunity/slack-notification-resource
+      tag: latest
+
+resources:
+  - name: pay-infra
+    type: git
+    icon: github
+    source:
+      uri: https://github.com/alphagov/pay-infra
+      branch: master
+      username: alphagov-pay-ci
+      password: ((github-access-token))        
+  - name: every-morning-at-five
+    icon: alarm
+    source:
+      start: "05:00"
+      stop: "05:30"
+      location: Europe/London
+    type: time
+  - name: every-morning-at-five-thirty
+    icon: alarm
+    source:
+      start: "05:30"
+      stop: "06:00"
+      location: Europe/London
+    type: time
+  - name: pay-ci
+    type: git
+    icon: github
+    source:
+      uri: https://github.com/alphagov/pay-ci
+      branch: master
+  - name: slack-notification
+    type: slack-notification
+    source:
+      url: https://hooks.slack.com/services/((slack-notification-secret))
+
+definitiions:
```

Comment:
```suggestion
definitions:
```

Reply:
I am surprised this works O_o

Not only works, but works! and doesn't error with a concourse validation or set-pipeline O_o whuuuuu

Reply:
Interestingly this is totally ignored whether I change it or not, so maybe concourse ignores all the top level keys it doesn't know

Reply:
Sorted both of them, thanks! 👍